src/Jackett.Common/Indexers/NCore.cs

```diff
@@ -51,280 +64,272 @@ public NCore(IIndexerConfigurationService configService, WebClient wc, Logger l,
-
             AddCategoryMapping("iso", TorznabCatType.PCISO, "Program Prog/ISO");
             AddCategoryMapping("misc", TorznabCatType.PC0day, "Program Prog/RIP");
             AddCategoryMapping("mobil", TorznabCatType.PCPhoneOther, "Program Prog/Mobil");
-
             AddCategoryMapping("ebook_hun", TorznabCatType.Books, "Könyv eBook/HU");
             AddCategoryMapping("ebook", TorznabCatType.Books, "Könyv eBook/EN");
         }
 
+
         public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
         {
             LoadValuesFromJson(configJson);
-
             if (configData.Hungarian.Value == false && configData.English.Value == false)
-                throw new ExceptionWithConfigData("Please select atleast one language.", configData);
-
+                throw new ExceptionWithConfigData("Please select at least one language.", configData);
             var loginPage = await RequestStringWithCookies(LoginUrl, string.Empty);
-            var pairs = new Dictionary<string, string> {
-                { "nev", configData.Username.Value },
-                { "pass", configData.Password.Value },
-                { "ne_leptessen_ki", "1"},
-                { "set_lang", "en" },
-                { "submitted", "1" },
-                { "submit", "Access!" }
+            var pairs = new Dictionary<string, string>
+            {
+                {"nev", configData.Username.Value},
+                {"pass", configData.Password.Value},
+                {"ne_leptessen_ki", "1"},
+                {"set_lang", "en"},
+                {"submitted", "1"},
+                {"submit", "Access!"}
             };
-
             if (!string.IsNullOrEmpty(configData.TwoFactor.Value))
                 pairs.Add("2factor", configData.TwoFactor.Value);
-
             var result = await RequestLoginAndFollowRedirect(LoginUrl, pairs, loginPage.Cookies, true, referer: SiteLink);
-            await ConfigureIfOK(result.Cookies, result.Content != null && result.Content.Contains("profile.php"), () =>
-            {
-                var parser = new HtmlParser();
-                var dom = parser.ParseDocument(result.Content);
-                var messageEl = dom.QuerySelector("#hibauzenet table tbody tr");
-                var msgContainer = messageEl.Children[1];
-                var errorMessage = msgContainer != null ? msgContainer.TextContent : "Error while trying to login.";
-                throw new ExceptionWithConfigData(errorMessage, configData);
-            });
-
+            await ConfigureIfOK(
+                result.Cookies, result.Content?.Contains("profile.php") == true, () =>
+                {
+                    var parser = new HtmlParser();
+                    var dom = parser.ParseDocument(result.Content);
+                    var messageEl = dom.QuerySelector("#hibauzenet table tbody tr");
+                    var msgContainer = messageEl.Children[1];
```

Comment:
I think this is safe in case the query selector is null
var msgContainer = dom.QuerySelector("#hibauzenet table tbody tr")?.Children[1];